disasterinfosite/models.py

```diff
@@ -333,80 +333,89 @@ def getRelatedTemplate(self):
     def findSnuggetsForPoint(lat=0, lng=0, merge_deform = True):
         pnt = Point(lng, lat)
 
-######################################################
-# GENERATED CODE GOES HERE
-# DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
-# modelsGeoFilters
```

Comment:
This is the least gross way to do it. The alternative would be to put something in the view that knows which snuggets should go where and moves them around, but it would have the same problems- we'd have to have knowledge of which shapefiles were the main ones, and which were secondary.